src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentOrderNotApprovedDocumentService.java

```diff
@@ -44,10 +48,20 @@
                 coverLetter(caseDetails, authorisationToken),
                 notApprovedConsentOrder(caseData),
                 applicantReplyCoversheet(caseDetails, authorisationToken));
+
+            if (featureToggleService.isPrintGeneralOrderEnabled() && isPaperApplication(caseData)) {
```

Comment:
can we move this logic into either a private method or below the current conditional to remove code duplication?

Reply:
good idea, done 👍